### PR TITLE
chore: cut release 1.8

### PR DESCRIPTION
**Pull request by slonka**
Signed-off-by: slonka <slonka@users.noreply.github.com>

Cut the website for the 1.8.x release. Followed https://github.com/kumahq/kuma-website/blob/master/WRITING-DOCUMENTATION.md#cutting-a-new-release

Did you sign your commit? [Instructions](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md#sign-your-commits) yes

Have you read [Contributing guidelines](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md)? yes

### CODE REVIEW

File: docs/docs/1.7.x/versions.json [hunk 1]
```diff
@@ -1,5 +1,4 @@
 {
   "helm": ["1.7.1"],
-  "kuma": ["1.7.1"],
-  "latest": true
```
Comment (slonka) [inline docs/docs/1.7.x/versions.json:4]: removed this

File: docs/docs/1.8.x/versions.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "helm": ["1.8.0"],
+  "kuma": ["1.8.0"],
+  "latest": true
```
Comment (slonka) [inline docs/docs/1.8.x/versions.json:4]: added this

**Comment by slonka** [comment]
Everything is copied with the exception of two things I mentioned in the comments.

File: docs/docs/1.8.x/versions.json (hunk 2 above)
Comment (slonka) [reply docs/docs/1.8.x/versions.json]: not yet

**Review by kleinfreund — APPROVED** [review]
I’ve ran this locally and it all seems to work. You’ve intentionally kept 1.7.x as the latest and that works, too.

**Pull request merged**